app/views/layouts/_head.html.haml

```diff
@@ -0,0 +1,29 @@
+%link{ rel: "preconnect", href: "https://fonts.googleapis.com" }
+%link{ rel: "preconnect", href: "https://fonts.gstatic.com", crossorigin: true }
+%link{ rel: "preconnect", href: Exercism.config.website_assets_host, crossorigin: true }
+
+/ We always want this rendered at the start.
+= stylesheet_link_tag "website", "data-turbo-track": "reload"
+= stylesheet_link_tag "https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Source+Code+Pro:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,900&display=swap"
+- if Rails.env.production?
+  %script{ defer: true, src: "/usage/js/script.js", data: { domain: "exercism.org", api: "/usage/api/event" } }
+
+- if landing_page?
+  - content_for :deferred do
+    = javascript_include_tag('landing', type: :module, crossorigin: :anonymous, 'data-turbo-track': 'reload', 'data-turbo-eval': false, defer: true)
+    = stylesheet_link_tag "internal", rel: "prefetch stylesheet", as: :style
+    = stylesheet_link_tag "application", rel: "prefetch stylesheet", as: :style
+    %link{ href: asset_path("core.js"), rel: "prefetch", as: :script }
+
+- else
+  = stylesheet_link_tag "internal", "data-turbo-track": "reload"
+  = stylesheet_link_tag "application", "data-turbo-track": "reload"
+  = javascript_include_tag('core', type: :module, crossorigin: :anonymous, 'data-turbo-track': 'reload', 'data-turbo-eval': false)
+  - js_packs.each do |pack|
+    = javascript_include_tag(pack, type: :module, crossorigin: :anonymous, 'data-turbo-track': 'reload', 'data-turbo-eval': false)
+
+- content_for :deferred do
+  / Prefetch the deferred packs which are needed on other pages
+  - deferred_js_packs.each do |pack|
+    %link{ href: asset_path("#{pack}.js"), rel: "prefetch", as: "script" }
```

Comment:
Having this here at the top level means it is also included for the non-landing pages. Currently, that means that on those pages, the `internal.js` bundle is prefetched. It might be slightly overkill to prefetch that library on all other pages 🤷 

Speaking of which, I see that we're including the `internal.css` stylesheets but are never including the `internal.js` bundle (except for prefetching). Is that correct?

Reply:
We can remove the manual internal.js prefetching as that's covered in the js_packs and deferred_js_packs bit.